maildir-utils's configure script spuriously rejects Xapian releases
with multidigit subminor versions, as we currently have pretty much
everywhere:

  checking for xapian-config... xapian-config
  configure: error: *** xapian version >= 1.2 needed, but version 1.2.10 found.
  make: *** [build-stamp] Error 1

Upstream commit

https://github.com/djcb/mu/commit/1dc44228d50f680d31080c760f8688e9675d6526

addresses the problem, though its description is off; could you please
apply it?
---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: maildir-utils.git/configure.ac
===================================================================
--- maildir-utils.git.orig/configure.ac	2012-05-13 12:02:08.996315355 +0900
+++ maildir-utils.git/configure.ac	2012-05-13 12:06:01.691160489 +0900
@@ -140,7 +140,7 @@
    *** is in your PATH.])],
    [xapian_version=$($XAPIAN_CONFIG --version | sed -e 's/.* //')])
 AS_CASE([$xapian_version],
-	[1.[[2-9]].[[0-9]]], [],
+	[1.[[2-9]].[[0-9]]*], [],
 	[AC_MSG_ERROR([*** xapian version >= 1.2 needed, but version $xapian_version found.])])
 
 XAPIAN_CXXFLAGS="`$XAPIAN_CONFIG --cxxflags`"
